### DISCUSSION TITLE
how to validate date min and max

### DISCUSSION THREAD

**Discussion by julio3266** [react-hook-form/react-hook-form#8723] [General]
I'm trying to create a validation for the input type for date, using the yup resolver. But this problem is that i need have the validation for date min max. As  in example[ on code sandbox](https://codesandbox.io/s/react-hook-form-js-forked-gbsf9?file=/src/App.tsx)

**Comment by Moshyfawn** [comment]
Pretty sure it's related to your latest question about date range validation.. #8736

**Reply by Moshyfawn** [reply]
https://github.com/react-hook-form/react-hook-form/discussions/8736#discussioncomment-3297359
